extension/view/js/translation-notification-fxtranslations.js

```diff
@@ -252,17 +252,17 @@ window.MozTranslationNotification = class extends MozElements.Notification {
     const langName = Services.intl.getLanguageDisplayNames(undefined, [lang,])[0];
 
     // set the label and accesskey on the menuitem.
-    const bundle = Services.strings.createBundle("chrome://browser/locale/translation.properties",);
+
     let item = this._getAnonElt("neverForLanguage");
-    const kStrId = "translation.options.neverForLanguage";
+    let label = this.translationNotificationManager.localizedLabels.neverForLanguageLabel;
+
     item.setAttribute(
       "label",
-      bundle.formatStringFromName(`${kStrId}.label`, [langName]),
+      label.replace("%S", langName),
```

Comment:
This is flimsy, but as far as I understand this code doesn't have access to browser.i18n functions